process/interface.go

```diff
@@ -200,14 +200,12 @@ type BlockProcessor interface {
 	RevertAccountState()
 	PruneStateOnRollback(currHeader data.HeaderHandler, prevHeader data.HeaderHandler)
 	RevertStateToBlock(header data.HeaderHandler) error
-	CreateNewHeader(round uint64) data.HeaderHandler
+	CreateNewHeader() data.HeaderHandler
+	UpdateEpochStartTriggerRound(round uint64)
```

Comment:
as the decode functions do not exists any more. CreateNewHeader can be reverted to how it is in development. No need for new method on blockprocessor interface.

Reply:
Done